tensorflow/lite/c/CMakeLists.txt

```diff
@@ -36,6 +36,30 @@ add_subdirectory(
 
 set(CMAKE_CXX_STANDARD 11)
 
+if (CMAKE_SIZEOF_VOID_P EQUAL 4)
```

Comment:
If this is only needed for Windows build, let's unify this with the condition of line #155

Reply:
Sure thing, did you mean line 45 though?

Reply:
There is a block with `if(CMAKE_SYSTEM_NAME MATCHES "Windows")`
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/CMakeLists.txt#L155
I think this logic should be inside of the block.

Reply:
Sure thing I could also insert it there. Probably inside the block with ` if(MSVC)` since i'm not sure if other compilers would support/require the same flag changes https://github.com/tensorflow/tensorflow/blob/704610e1d21288482bf77923b387f9bb6c119318/tensorflow/lite/CMakeLists.txt#L162
Was just confused because my PR was originally for the Lite C API CMakeLists, but it could also be useful for the Lite main CMakeLists, since the same build issue occurred las time I tried to build it. Lately i've being building only the C API DLL though.

Should we place it on both CMakeLists so?

Reply:
I think we only need to update the main CMakeLists.txt file. Could you verify?
If it doesn't work, let's just wrap it with `if(CMAKE_SYSTEM_NAME MATCHES "Windows")`

Reply:
I could be building it "wrong" but was directly calling the lite/c CMakeLists.txt from the command line, using cmake+msvc and building only the c api libraries. But for sure I can verify the update on the main one and check if it works. Will post it back here soon